File: DataLayer/Entities/PersonImage.cs
using System.IO;
using DataLayer.Entities.Common;

namespace DataLayer.Entities
{
    public class PersonImage : EntityBase, IImage
    {
        public int? PersonId { get; set; }
        public Person Person { get; set; }
        public string Name { get; set; }
        public string Thumbnail { get; set; }
        public string Url { get; set; }
        public string GetPath()
        {
            return $"{nameof(PersonImage)}/{Id}";
        }
    }
}

File: dataLayer/entities/person_image.py
from sqlalchemy import Column, String, Integer, Date, ForeignKey

from dataLayer.database_connection import Base


class PersonImage(Base):
    __tablename__ = 'PersonImage'

    id = Column(Integer, primary_key=True)
    name = Column('Name', String)
    thumbnail = Column("Thumbnail", String)
    person_id = Column("PersonId", Integer, ForeignKey("Person.Id"))

    def __init__(self, name, person_id):
        self.name = name
        self.person_id = person_id
